R/add_totals.R

```diff
@@ -1,9 +1,67 @@
+#' @title Append a totals row and/or column to a data.frame.
+#'
+#' @description
+#' This function excludes the first column of the input data.frame, assuming it's a descriptive variable not to be summed.  It also excludes other non-numeric columns.
+#'
+#' @param dat an input data.frame with at least one numeric column.
+#' @param which one of "row", "col", or \code{c("row", "col")} 
+#' @param fill if there are multiple non-numeric columns, what string should fill the bottom row of those columns?
+#' @param na.rm should missing values (including NaN) be omitted from the calculations?
+#' @return Returns a data.frame augmented with a totals row, column, or both.
+#' @export
+#' @examples
+#' library(dplyr) # for the %>% pipe
+#' mtcars %>%
+#'   crosstab(am, cyl) %>%
+#'   add_totals()
+
+
+add_totals <- function(dat, which = c("row", "col"), fill = "-", na.rm = TRUE){
```

Comment:
I prefer `adorn_totals` as this is something that is primarily intended to modify a `tabyl`. I think that `adorn_crosstab` may eventually become the outlier in the `adorn_` approach. I think I've given this feedback already, though, so I assume that you have heard it and decided on verb_object approach to things that modify a tabyl.